### ISSUE TITLE
pottery imports fail because of `tests` imports

### ISSUE THREAD

**Issue by Zopieux** [brainix/pottery#35]
`pottery-0.36-py3-none-any.whl`

```pycon
>>> from pottery import RedisSet  # or other DS
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.4/site-packages/pottery/__init__.py", line 35, in <module>
    from .contexttimer import ContextTimer
  File "/home/user/.local/lib/python3.4/site-packages/pottery/contexttimer.py", line 13, in <module>
    from tests.base import run_doctests
ImportError: No module named 'tests.base'
```

**Comment by brainix** [comment]
Hi, @Zopieux.

Thank you for the bug report.  I apologize, and I've released a fix in version 0.37.  Please try with the new version?

Thanks again,
Raj

**Comment by brainix** [comment]
Hi, @Zopieux.

I believe that I've resolved this issue with these two commits:
https://github.com/brainix/pottery/pull/36/commits/770e9d8357b557f6ff681852a9465d3f8f655815
https://github.com/brainix/pottery/pull/36/commits/c252179b2150ec2dc083dc2fabcf668a7c054985

Please let me know if you have any further problems, and thanks!
Raj
